packages/code-studio/src/plugins/PluginUtils.tsx

```diff
@@ -8,17 +8,20 @@ const log = Log.module('PluginUtils');
 class PluginUtils {
   /**
    * Load a component plugin either specified in the REACT_APP_COMPONENT_TABLE_PLUGINS environment variable, or from the server if it's not internal.
-   * @param {string} pluginName Name of the table plugin to load
-   * @returns {JSX.Element} A lazily loaded JSX.Element from the plugin
+   * @param pluginName Name of the table plugin to load
+   * @return A lazily loaded JSX.Element from the plugin
```

Comment:
Missing 's' on @returns
```suggestion
   * @returns A lazily loaded JSX.Element from the plugin
```